doc/guide.rst

```diff
@@ -691,6 +691,9 @@ The argument name is transliterated like this:
   If (a boolean argument) ``--quiet`` is passed, value of ``_arg_quiet`` is set to ``on``.
   Conversely, if ``--no-quiet`` is passed, value of ``_arg_quiet`` is set to ``off``.
 
+* Provided optional arguments set ``_supplied_<arg-name>``.
```

Comment:
I suggest explicitly standardizing to "set to 1", and mentioning it here.

Reply:
Done.